src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,29 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
+
+	public String value;
+	protected boolean isPrivate;
+
+	@Override
+	public String toString() {
+	    return value;
+	}
+
+	@Override
+	public boolean equals(Object other) {
```

Comment:
Beware of the issue we discussed in tutorial, i.e. subclasses of different type may be "equal" if they contain the same value by this code.